src/jit/flowgraph.cpp

```diff
@@ -13490,13 +13490,15 @@ bool Compiler::fgOptimizeSwitchBranches(BasicBlock* block)
         GenTree* switchVal = switchTree->gtOp.gtOp1;
         noway_assert(genActualTypeIsIntOrI(switchVal->TypeGet()));
 
+#ifndef LEGACY_BACKEND
```

Comment:
Can you add a comment explain why you ifdef-ed this:
(i.e. The only new change is that JMPTABLE has to be #ifdef'd based on LEGACY_BACKEND, or the 32-bit build will fail when MEASURE_NODE_SIZE is enabled.)